spec/pdc/v1/global_component_contact_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+describe PDC do
+  before do
+    VCR.insert_cassette fixture_name
+  end
+
+  after do
+    VCR.eject_cassette
+  end
+
+  let(:contact) { PDC::V1::GlobalComponentContact }
+
+  it 'must has compose' do
+    contact.where(component: 'vlgothic-fonts').first.contact.must_be_instance_of OpenStruct
+  end
+
+  describe 'count' do
+    it 'must return number of resources' do
+      count = contact.count
+      count.must_equal 12546
+    end
+
+    it 'works with where' do
+      count = contact.where(component: 'vlgothic-fonts' ).count
+      count.must_equal 3 # as there is 3 fields per entry
+    end
+
+    it 'can find global contact by component name' do
+      count = contact.where(component: 'vlgothic-fonts').count
+      count.must_equal 3
+    end
+  end # count
+
+  describe '#contact' do
+    it 'returns contact for component' do
+      vf = contact.where(component: 'vlgothic-fonts').first
+      vf.contact.wont_be_nil
+    end
+
+    it 'returns a contact email' do
+      vf = contact.where(component: 'vlgothic-fonts').first
+      vf.contact.email.must_equal 'qe-i18n-bugs@redhat.com'
+    end
+
+    it 'returns a mail_name' do
+      vf = contact.where(component: 'vlgothic-fonts').first
+      vf.contact.mail_name.must_equal 'I18N'
+    end
+
+    it 'returns a gloabl contact id' do
+      vf = contact.where(component: 'vlgothic-fonts').first
+      vf.contact.id.must_equal 5
+    end
+  end # contact
+
+  describe '#role' do
+    it 'returns contact role' do
+      bash_contact = contact.where(component: 'bash').first
+      bash_contact.role.wont_be_nil
+      bash_contact.role.must_equal 'QE_Group'
+    end
+  end # role
```

Comment:
same with above, pronto error

Reply:
Actually, out of curiosity, @simozhan where do you see the pronto error? I've checked the last(?) run:

https://travis-ci.org/product-definition-center/pdc-ruby-gem/builds/312237759?utm_source=github_status&utm_medium=notification

and couldn't find any errors.

Reply:
@romanofski I test it locally with rubocop spec/pdc/v1/global_component_contact_spec.rb 
Yeah. You are right. There is no errors in the travis build. Actually I have the same question why travis not pop them out. These codes should be in the diff, why no errors